### PR TITLE
Closes #1214: Add CHANGELOG for v3.0

### LINKED ISSUE #1214: Append additional changes to the CHANGELOG for v3.0
I don't think we've been adding to the CHANGELOG so we should go through after code freeze to see what additional functionality has changed and add that to the CHANGELOG.

### PR DESCRIPTION
**Pull request by dnarcese**
The date will still need to be added once it is known. Not sure if this should be merged before that is added.
Also, I preemptively changed the link for v3.0 even though it won't show anything right now.

### CODE REVIEW

**Comment by dnarcese** [comment]
@mcomella Should I just leave the link as it was, and keep it as unreleased for now?

**Comment by mcomella** [comment]
> @mcomella Should I just leave the link as it was, and keep it titled as 'Unreleased' for now?

I'd leave it unreleased: we can change it to v3.0 when we bump the build.gradle version.

**Comment by mcomella** [comment]
Then again, this is probably fine; I've been waiting for the version bump on APT Github but I suppose it's not necessary.

Can you file a follow-up to add a release date (maybe in the next Sprint?)?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,13 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
```
Comment (mcomella) [inline CHANGELOG.md]: You should keep the unreleased section here so we can continue to add items underneath it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -74,7 +75,7 @@ The CHANGELOG entries for the releases listed below this were added retroactivel
 ## [1.0] - 2017-12-20
 *Initial release! A browser including home tile shortcuts.*
 
-[Unreleased]: https://github.com/mozilla-mobile/firefox-tv/compare/v2.2...HEAD
```
Comment (mcomella) [inline CHANGELOG.md]: Same, this should remain and point from v3.0 to HEAD.

**Comment by mcomella** [comment]
Oh, maybe we want to add "startup time improvements" too: https://github.com/mozilla-mobile/firefox-tv/issues/1101

**Pull request merged**